include/opencl_utils.h

```diff
@@ -115,7 +115,7 @@ class OpenCL {
 				sort(times.begin(), times.end());
 				auto median = times[times.size() / 2];
 				// Save result to file
-				File::add_time(run.hash, median, local_size);
+				File::add_time(run.hash, median, cl::NDRange(run.glob1, run.glob2, run.glob3),local_size);
```

Comment:
Because the kernel is queued with the localsize NDrange variable local_size.
For the global range run.glob etc is used to queue.
Is there a scenario where the local_size and the run.local_size is different?